### DISCUSSION TITLE
Issue with full rollups, post version upgrade

### DISCUSSION THREAD

**Discussion by RtimmFF** [jamessimone/apex-rollup#436] [General]
Hi there,

We're seeing an issue after upgrading to v1.5.58 with getting incorrect results when we execute multiple rollups from the same Object via the Rollup App interface for full rollups or for running via record-triggered Flows for Object with multiple rollups.

We don't see the errors when we run each full rollup separately, just when more than one is selected in the same execution.  I can send you a doc with details and some log entries.

**Comment by jamessimone** [comment]
@RtimmFF I will take a look through what you've sent over. Just a head's up, response times may be a bit slower than usual as I'm working to release `v1.5.62` at the moment and I was out last week.

**Reply by RtimmFF** [reply]
Hi James, 

Can you give me an update on what the timeline looks like for your release plan and any insights or next steps you suggest on this item? Thanks!

**Reply by jamessimone** [reply]
@RtimmFF can you re-send over what you had originally? Sorry, looking through my email and I'm afraid I'm having trouble locating our prior communication about this. Please ensure you're on the latest version (`v1.5.63`, presently). 

I'm still working through my backlog from having been out, but I will try to provide an update for you this week.

**Reply by RtimmFF** [reply]
OK, thanks, I'll resend.

**Reply by RtimmFF** [reply]
Hi James,

Were you able to take a look at this? We haven't been able to deploy this as originally intended. Just hit a hold on temp solution with executing each rollup separately. 

Thanks !

**Reply by jamessimone** [reply]
Have you tried setting the `Should Skip Resetting Parent Fields` flag to true on your Rollup Control org default record? It's been five months since 1.5.43 was released; I'm happy to try to triage this issue for you, but without knowing which release actually introduced this regression for you, it will be difficult for me to assist further.

I recognize that it's time consuming to go through and install 1.5.44 - 1.5.63 (and I'm about to release 1.5.64) but because the logs you've sent me are already several versions behind, and because there have been 20 versions between then and now, I don't have enough info to assist.
